test/Microsoft.AspNet.Mvc.IntegrationTests/MutableObjectModelBinderIntegrationTest.cs

```diff
@@ -550,6 +551,9 @@ private class Person3
             var operationContext = ModelBindingTestHelper.GetOperationBindingContext(request =>
             {
                 request.QueryString = new QueryString("?parameter.Customer.Name=bill");
+
+                // This is set so that the input formatter does not add an error to model state.
```

Comment:
Confusing comment.  Do you mean this avoids an error unrelated to the test scenarios?